mriqc/workflows/diffusion/base.py

```diff
@@ -139,10 +144,7 @@ def dmri_qc_workflow(name="dwiMRIQC"):
     drift = pe.Node(CorrectSignalDrift(), name="drift")
 
     # 2. Generate B0 reference
-    dwi_reference_wf = init_epi_reference_wf(
-        omp_nthreads=config.nipype.omp_nthreads,
-        name="dwi_reference_wf",
-    )
+    dwi_reference_wf = init_dmriref_wf()
```

Comment:
Workflows should have the same name as the variables that point to them.

```suggestion
    dwi_reference_wf = init_dmriref_wf(name="dwi_reference_wf")
```